docs/getting-started-guides/kubeadm.md

```diff
@@ -177,12 +177,43 @@ You must install a pod network add-on so that your pods can communicate with eac
 
 **It is necessary to do this before you try to deploy any applications to your cluster, and before `kube-dns` will start up. Note also that `kubeadm` only supports CNI based networks and therefore kubenet based networks will not work.**
 
-Several projects provide Kubernetes pod networks using CNI, some of which 
-also support [Network Policy](/docs/user-guide/networkpolicies/). See the [add-ons page](/docs/admin/addons/) for a complete list of available network add-ons.
-
-You can install a pod network add-on with the following command: 
-
-    # kubectl apply -f <add-on.yaml>
+Several projects provide Kubernetes pod networks using CNI, some of which also support [Network Policy](/docs/user-guide/networkpolicies/).
+
+You must select a network addon from the drop-down menu below, and correct instructions will appear below.
+
+See the [add-ons page](/docs/admin/addons/) to learn more about each of the available network add-ons.
+
+<script>
+  var networkAddons = {
+    "Calico": "kubectl apply -f \"http://docs.projectcalico.org/v2.0/getting-started/kubernetes/installation/hosted/calico.yaml\"",
```

Comment:
@caseydavenport I think we should try to avoid having a version number in the URL.  Can we have a "latest" or similar that corresponds to the latest stable release?